fw-child/template/query/learn-item.php

```diff
@@ -1,25 +1,42 @@
+<?php
+// Initialize current language.
+$current_lang = 'en';
+
+if ( isset( $item['lang'] ) && in_array( $item['lang'], array( 'en', 'fr' ), true ) ) {
+	$current_lang = $item['lang'];
+}
+
+$native_excerpt = apply_filters ( 'the_content', custom_excerpt ( 20, $item['id'] ) );
```

Comment:
The `custom_excerpt()` function already applies the `'the_content'` filter of the content it returns, so it would not be required here.